Base/res/html/misc/clip.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
```

Comment:
The prefix for this commit should be `Base: ` not `LibWeb: `, also I'd maybe call it a "test page" :^)